harness/scripts/destroy.sh.twig

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+function destroy() {
+  run docker-compose down --rmi local --volumes --remove-orphans
+
+  if [ "{{ @('docker.experimental.multiplatform_build.enabled') }}" == "yes" ]; then
+    builder=$(docker buildx inspect | grep Name: | head -n 1 | awk '{print $2}')
```

Comment:
```bash
docker buildx inspect | grep Name: | head -n 1 | awk '{print $2}'
default
```
"default" != "" but default and desktop-linux shouldn't be deleted

Reply:
oops, I did have a grep in here for `docker-container` driver which would have prevented that, not sure how I missed it in the final implementation, but won't matter now anyway as we will use a named builder 👍🏻